views/js/previewer/runner.js

```diff
@@ -39,16 +38,16 @@ define([
      * @param {Number|String} [config.height] - The height in pixels, or 'auto' to use the container's height
      * @param {Boolean} [config.options.fullPage] - Force the previewer to occupy the full window.
      * @param {Booleanh} [config.options.readOnly] - Do not allow to modify the previewed item.
-     * @param {jQuery|HTMLElement|String} container - The container in which renders the component
      * @param {Function} [template] - An optional template for the component
      * @returns {previewer}
      */
-    return function previewerFactory(config = {}, container, template) {
+    return function previewerFactory(container, config = {}, template = null) {
 
-        return runnerComponentFactory(container || $(document.body), config, template || runnerTpl)
+        return runnerComponentFactory(container, config, template || runnerTpl)
             .on('render', function() {
-                this.setState('fullpage', config.options.fullPage);
-                this.setState('readonly', config.options.readOnly);
+                const options = this.getConfig().options;
```

Comment:
Just a small improvement:
`const {fullPage, readOnly} = this.getConfig().options;`

Reply:
Right, that would be less verbose.